### PR TITLE
Volcano Plot for RUMP

### PR DESCRIPTION
**Pull request by xkcococo**
I have not modify main.nf to integrate my new code to RUMP.

### CODE REVIEW

File: rump/Volcano_Plot.R [hunk 1]
```diff
@@ -0,0 +1,130 @@
+#Library your packages
```
[the remark this answers is not in the document]
Comment (xkcococo) [reply rump/Volcano_Plot.R:17]: Hi, I have already added a project overview information.

File: rump/Volcano_Plot.R [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    ay = -40
+  )
+}
+
+# make the Plot.ly plot
+p <- plot_ly(data = sample_data, x = ~Fold, y = ~-log10(FDR),
+             mode = "markers", color = ~group) %>%
+  layout(title = "Volcano Plot") %>%
+  layout(annotations = a)
+htmlwidgets::saveWidget(p, file = opt$volcano_plot)
```
[the remark this answers is not in the document]
Comment (xkcococo) [reply rump/Volcano_Plot.R]: Hi, I converted the output from html to png.

File: rump/Volcano_Plot.R [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+library(processx)
+library(optparse)
+library(cmmr)
+
+# 20 Digits Precision Representation
+options(scipen = 20)
+# If shows every warning
+options(warn = -1)
+
+setwd("/Users/xu.ke/Dropbox (UFL)/Research/Active/20200707_RUMP/DATA0")
```
[the remark this answers is not in the document]
Comment (xkcococo) [reply rump/Volcano_Plot.R]: Hi, I added a # at the front of this command line to avoid running.

**Review by xkcococo — COMMENTED** [review]
Modified codes by checking with lintr

**Review by XinsongDu — APPROVED** [review]

**Pull request merged**